frontend/src/components/CardTitle/CardTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Typography,} from "@material-ui/core";
+import {APP_NAME} from "../../constants/constants";
+import {VerticalSpace} from "../Shared/VerticalSpace"
+import {Centered} from "../Shared/Centered";
+import styled from "styled-components";
+
+export type TitleProps = {
```

Comment:
To props zazwyczaj odpowiada nazwie komponentu, czyli np. CardTitleProps.

Reply:
Ok